cmake/gsInstall.cmake

```diff
@@ -163,10 +163,23 @@ install(FILES
 #install(EXPORT gismoTargets DESTINATION
 #  "${CMAKE_INSTALL_DIR}" COMPONENT devel)
 
+# Produce pkg-config file
+configure_file ("${PROJECT_SOURCE_DIR}/gismo_lib.pc.in"
+                "${PROJECT_BINARY_DIR}/gismo.pc" @ONLY)
+
 else(GISMO_BUILD_LIB)
-   message ("Configure with -DGISMO_BUILD_LIB=ON to compile the library")
+  message ("Configure with -DGISMO_BUILD_LIB=ON to compile the library")
+
+# Produce pkg-config file
+  configure_file ("${PROJECT_SOURCE_DIR}/gismo_nolib.pc.in"
+                  "${PROJECT_BINARY_DIR}/gismo.pc" @ONLY)
 endif(GISMO_BUILD_LIB)
 
+# Install pkg-config file
+install(FILES "${PROJECT_BINARY_DIR}/gismo.pc"
+  DESTINATION "lib/pkgconfig/"
```

Comment:
Thanx for this addtion!

Is this path platform independent?
Maybe we can use 

${LIB_INSTALL_DIR}/pkgconfig/

instead ?

Reply:
I totally agree with you! I don't really know.

In my system, SDL, OGRE and CGAL they all put `.cmake` files in `lib/cmake/xxx/`. And they all put `.pc` files in `lib/pkgconfig/`.

But I have no idea about specifications and recommendations. I will research a little.

Reply:
> ${LIB_INSTALL_DIR}/pkgconfig/
> instead ?

Of course! I was a little confused because I was under the impression `LIB_INSTALL_DIR` was `lib/gismo`. Sorry! `LIB_INSTALL_DIR` is fine. Others usually use `CMAKE_INSTALL_LIBDIR`.

> Is this path platform independent? Maybe we can use

Just for the record, it seems FreeBSD does not like this very much... SDL does it like this:
https://github.com/libsdl-org/SDL/blob/26728c321fa9ba5300e90fc14f32fa1f974b9ab8/CMakeLists.txt#L3448-L3453

Do we have a way to detect FreeBSD, or shall I use `LIB_INSTALL_DIR`?

Reply:
OK, thnx for all the research.
You may detect FreeBSD as
if( ${CMAKE_SYSTEM_NAME} MATCHES "FreeBSD" )

Reply:
Done.

I am very sorry I used to commits... First I had used a variable named `xxx_INSTALLDIR`, then I got it changed to `xxx_INSTALL_DIR`.